app/controllers/api/v1/amusement_parks_controller.rb

```diff
@@ -1,9 +1,23 @@
-class Api::V1::AmusementParksController < ApplicationController
+class Api::V1::AmusementParksController < ApiController
+  protect_from_forgery unless: -> {request.format.json?}
+
   def index
     render json: AmusementPark.all
   end
 
   def show
-    render json: AmusementPark.find(params[:id])
+    render json: {
+      amusement_park: AmusementPark.find(params[:id]),
+      reviews: serialized_reviews,
+      rides: serialized_rides
+    }
+  end
+
+  def serialized_reviews
+    ActiveModel::Serializer::ArraySerializer.new(AmusementPark.find(params[:id]).reviews, each_serializer: ReviewSerializer)
```

Comment:
what happens if the `AmusementPark` is not found? Also, could you use the one that is already found in the body of the `show`?